provider/confluence/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@openctx/provider-confluence",
+  "private": false,
+  "version": "0.0.6",
```

Comment:
version seems high for something new. Although fine if you have already hit pnpm publish.